pkg/common/common.go

```diff
@@ -176,12 +185,13 @@ func ConvertConfigToV3(config *Config) *OpenAPIV3Config {
 		GetOperationIDAndTags:          config.GetOperationIDAndTags,
 		GetOperationIDAndTagsFromRoute: config.GetOperationIDAndTagsFromRoute,
 		GetDefinitionName:              config.GetDefinitionName,
+		Definitions:                    config.Definitions,
```

Comment:
I must be mis-reading, but what sets that definitions? I can only see it read, I can't find where it's written.

Reply:
k/k config setup

Reply:
Ah, yeah, right of course